### ISSUE TITLE
deploy api

### ISSUE THREAD

**Issue by abz53378** [CannerCMS/cannercms#14]

**Comment by abz53378** [comment]
## dataDidChange
```js
({
  [key]: boolean | Array<string>
}) => void
```
## CMS.deploy
## CMS.reset
## afterDeploy
```js
{{
 key: string,
 id: ?string 
}} => void
```
## hideButtons
```js
boolean
```

**Comment by abz53378** [comment]
fixed in b0af5b12152eb4644b4ab4d8a421e581aa1c49ef and 5ac4d9cf9583848854f968191de8d0a9121448a8
